### PR TITLE
Add note about blocklist distribution

### PR DESCRIPTION
**Pull request by JohnMaguire**
This closes out the documentation task https://github.com/slackhq/nebula/issues/304.

### CODE REVIEW

File: docs/config/pki.mdx [hunk 1]
```diff
@@ -54,9 +54,16 @@ hosts.
 
 ## pki.blocklist
 
-The `blocklist` contains a list of individual host certificates to ignore. In the case a host’s credentials are stolen
-or compromised, this allows us to block connectivity from a host, even if it is signed by a certificate authority we
-trust.
+:::note
+
+The blocklist is _not_ distributed via Lighthouses. To ensure access to your entire network is blocked you must distribute
```
Comment (jasikpark) [inline docs/config/pki.mdx:59]: maybe be more explicit about it being a static config value to update?

**Review by jasikpark — APPROVED** [review]

File: docs/config/pki.mdx (hunk 1 above)
Comment (JohnMaguire) [reply docs/config/pki.mdx:59]: I'm not sure I understand. What do you mean by static config value to update?

Comment (jasikpark) [reply docs/config/pki.mdx:59]: "you must distribute" = "update the configuration file on each host & HUP them" I guess I was arguing be more explicit

Comment (JohnMaguire) [reply docs/config/pki.mdx:59]: Let's leave it like this for now and add a separate section on updating configs (including how to reload them.) Then we can link to it from here if we think it's helpful. But in general I'm trying to focus on a specific config item (`pki.blocklist`) and not get into the weeds on how the config itself works. :)

**Pull request merged**